src/views/virtualmachines/list/components/VMIP/VMIP.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+  V1VirtualMachineInstance,
+  V1VirtualMachineInstanceNetworkInterface,
+} from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { Button, ButtonVariant, Popover, PopoverPosition } from '@patternfly/react-core';
+
+type VMIPProps = {
+  vmi: V1VirtualMachineInstance;
+};
+
+const getVMIIPAddresses = (vmi: V1VirtualMachineInstance): string[] => {
+  return [
+    ...new Set(
+      vmi.status?.interfaces
+        ?.filter((i) => !!i.name) // get IPs only for named interfaces because Windows reports IPs for other devices like Loopback Pseudo-Interface 1 etc.
+        .flatMap((i: V1VirtualMachineInstanceNetworkInterface) => {
+          const ips: string[] = [];
+          if (i.ipAddress) {
+            ips.push(i.ipAddress.trim());
+          }
+          if (i.ipAddresses) {
+            ips.push(...i.ipAddresses.map((ip) => ip.trim()));
+          }
+          return ips;
+        })
+        .filter((ip) => ip && ip.length > 0),
+    ),
+  ];
+};
+
+const VMIP: React.FC<VMIPProps> = ({ vmi }) => {
+  const { t } = useTranslation('plugin__kubevirt-plugin');
+
+  if (!vmi) {
+    return null;
+  }
+
+  const data: string[] = getVMIIPAddresses(vmi);
+  const isMoreThenOneIp = data.length > 1;
+  return (
+    <>
+      <>{data?.[0]}</>
+      {isMoreThenOneIp && (
+        <Popover
+          headerContent={<>{t('kubevirt-plugin~IP Addresses ({{ips}})', { ips: data.length })}</>}
+          bodyContent={Array.isArray(data) && data?.map((ip) => <>{ip}</>)}
+          hasAutoWidth
+          position={PopoverPosition.top}
+        >
+          <Button variant={ButtonVariant.link}>
+            {t('kubevirt-plugin~+{{ips}} more', { ips: data.length - 1 })}
+          </Button>
+        </Popover>
+      )}
+    </>
+  );
+};
```

Comment:
```suggestion
const OnePlusItems: React.FC<OnePlusItemsProps> = ({ items: steing[], headerContent:  ..., className: ... }) => (
    <className ... >
      <>{items?.[0]}</>
      {items.length > 1 && (
        <Popover
          headerContent={headerContent}
          bodyContent={items.map((item) => <>{item}</>)}
          hasAutoWidth
          position={PopoverPosition.top}
        >
          <Button variant={ButtonVariant.link}>
            {t('kubevirt-plugin~+{{itemsListSize}} more', { itemsListSize: items.length - 1 })}
          </Button>
        </Popover>
      )}
    </>
 );
```
don't know about the component name, but his looks like a generic component

Reply:
+1 changed